test_infra/setup.py

```diff
@@ -0,0 +1,32 @@
+import setuptools
```

Comment:
Is this even required?

Reply:
Missed that one -- nope, not required.

Reply:
Just to be clear, I meant the entire file is not required not just the library. Just realized my comment was not obvious :P